src/Core/SettingsCollection.cpp

```diff
@@ -485,6 +485,12 @@ void SettingURI::deserialize(ReadBuffer & buf, SettingsBinaryFormat)
 IMPLEMENT_SETTING_ENUM(LoadBalancing, LOAD_BALANCING_LIST_OF_NAMES, ErrorCodes::UNKNOWN_LOAD_BALANCING)
 
 
+#define SPECIAL_SORT_ALGORITHM_NAMES(M) \
+    M(NOT_SPECIFIED, "not_specified") \
+    M(BITONIC_SORT, "bitonic_sort")
```

Comment:
Rename to `opencl_bitonic`. We could have cuda variant in future. And word `sort` is not needed cause it apears in setting name.